config/crd/bases/cluster.x-k8s.io_machinesets.yaml

```diff
@@ -18,7 +18,6 @@ spec:
     shortNames:
     - ms
     singular: machineset
-  preserveUnknownFields: false
```

Comment:
We likely don't want to drop this, it may require updating the kubebuilder annotation for MachineSet to set `crd:preserveUnknownFields=false` as per https://github.com/kubernetes-sigs/controller-tools/pull/362